CHANGELOG.md

```diff
@@ -5,6 +5,18 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [2.1.0] - 2023-04-04
+
+### Added
+
+- Adds optional `vintage_start_year` and `vintage_end_year` fields to `order` creation
+- Adds optional `vintage_start_year` and `vintage_end_year` fields to `order_line_item` create and update
+- Adds optional `vintage_start_year` and `vintage_end_year` fields to `inventory` creation
+- Adds `vintage_start_year` and `vintage_end_year` fields to `order` response
+- Adds `vintage_start_year` and `vintage_end_year` fields to `order_line_item` response
+- Adds optional `carrier_scac` field to `Patch::Estimate.create_road_shipping_estimate` method
+- Deprecates `Patch::Estimate.create_shipping_estimate` in favor of `Patch::Estimate.create_ecommerce_estimate`
```

Comment:
Note that there doesn't seem to be a related change in the code. Is it just behind the scenes?

Reply:
I believe it's in relation to https://github.com/patch-technology/patch/pull/5588. @holtbp merged his change that fixes the integration tests and we spoke offline on updating the docs